cmake/Modules/LocalSvt.cmake

```diff
@@ -57,6 +58,11 @@ else()
     set(CMAKE_OUTPUT_DIRECTORY_ORIG "${CMAKE_OUTPUT_DIRECTORY}")
     set(CMAKE_OUTPUT_DIRECTORY "${SVT_BINARY_DIR}" CACHE INTERNAL "")
 
+    # For now, this creates compilation issues:
+    # _deps/svt-build/libSvtAv1Enc.a: error adding symbols: file format not recognized
+    # clang: error: linker command failed with exit code 1 (use -v to see invocation)
+    set(SVT_AV1_LTO OFF)
```

Comment:
As a small nit, it might be possible to instead determine this based on the value of `CMAKE_INTERPROCEDURAL_OPTIMIZATION`.
However, this review comment might be unnecessary as it should be that if the user defines `CMAKE_INTERPROCEDURAL_OPTIMIZATION` for libavif, SVT-AV1 should also inherit it, and `SVT_AV1_LTO` should be a one-way variable (it will set `CMAKE_INTERPROCEDURAL_OPTIMIZATION` to `ON` if `SVT_AV1_LTO` is on and LTO is supported, it does not ever set it to `OFF`). So if the user does set `CMAKE_INTERPROCEDURAL_OPTIMIZATION`, this will only disable that LTO support check and not actually disable LTO.

Reply:
I wanted to add a proper option for libavif in a different PR but I can already check for CMAKE_INTERPROCEDURAL_OPTIMIZATION indeed.